src/ios/CDVCamera.m

```diff
@@ -402,6 +444,115 @@ - (NSString*)tempFilePath:(NSString*)extension
     return filePath;
 }
 
+// This is not absolute, but more mightNeedOrientationCorrection
+- (BOOL) needsOrientationCorrection:(UIImage*)image options:(CDVPictureOptions*)options
```

Comment:
We don't need a separate function for this, I think we can have these options 'inline' where needed instead of creating separate functions for them.